src/Altinn.App.Core/Internal/Pdf/PdfService.cs

```diff
@@ -243,6 +268,22 @@ private async Task<string> GetLanguage()
         return language;
     }
 
+    private string GetOverriddenLanguage()
+    {
+        // If the user has overridden the language from his profile by selecting a
+        // language directly in the form, then use the selected language.
+        if (_httpContextAccessor.HttpContext != null)
+        {
+            bool hasQueryLanguage = _httpContextAccessor.HttpContext.Request.Query.TryGetValue("lang", out StringValues queryLanguage);
+            if (hasQueryLanguage)
+            {
+                return queryLanguage.ToString();
+            }
+        }
+
+        return string.Empty;
```

Comment:
With nullable reference types, you can safely return `null` and the compiler will ensure that the null is handled properly.